771.py

```diff
@@ -0,0 +1,10 @@
+class Solution:
+    def numJewelsInStones(self, jewels: str, stones: str) -> int:
+        jewels_set = set(list(jewels))
+        jewels_num = 0
```

Comment:
`jewels_count` would be more accurate. There is a slight difference between a count and a number.